source/stylesheets/refills/unstyled/_vertical-tabs.scss

```diff
@@ -0,0 +1,53 @@
+.vertical-tabs-container {
+  $large-screen: em(640) !default;
+  $vertical-tabs-height: em(100);
+
+  @include clearfix;
+
+  .vertical-tabs {
+    display: none;
+
+    @include media($large-screen) {
+      display: inline;
+      float: left;
+      height: $vertical-tabs-height;
+      width: 20%;
+    }
+  }
+
+  .vertical-tab {
+    @include media($large-screen) {
+      display: block;
+
+      &.is-active {
+        color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.